app/assets/javascripts/authoring/eventDraw.js

```diff
@@ -280,7 +280,13 @@ function drawBottom(eventObj) {
     
     // the number of members
     addToTaskFromData(events.numMembers, eventObj, task_g);
-    
+
+    if( !in_progress && flashTeamsJSON["paused"] != true){
```

Comment:
I think this should be: if( !in_progress || (in_progress == true && flashTeamsJSON["paused"] != true))